app/views/session/new.erb

```diff
@@ -2,5 +2,5 @@
 <form action='/session/new' method ='post'>
   E-Mail: <input type='email' name='email'>
   Password: <input type='password' name='password'>
-  <input type='submit' value='Sign in'>
+  <input type='submit' id='sign_in' value='Sign in'>
```

Comment:
Agree that we shouldn't add ids for the sake of testing. Only if they are necessary for implementation.